google/apistore.go

```diff
@@ -54,3 +54,29 @@ func (c *APIStore) AddReader(bucket, object, entity string) error {
 	_, err := c.service.ObjectAccessControls.Insert(bucket, object, ac).Do()
 	return err
 }
+
+// AddBucketReader adds a writer of the bucket
```

Comment:
[nit] is the comment supposed to say `adds a reader of the bucket` rather than `writer`?

Reply:
yep, thanks for catching that